ethpm_types/source.py

```diff
@@ -91,6 +91,39 @@ class Source(BaseModel):
     """
     # TODO: Add `SourceId` type and use instead of `str`
 
+    def __repr__(self) -> str:
+        repr_id = "Source"
+        if self.checksum:
+            repr_id = f"{repr_id} {self.checksum.hash}"
```

Comment:
Should this be the file URI instead e.g. `self.urls[0]`?

Reply:
- [x] Make use and favor URI when available.